retrieverdash/dashboard_script/dashboard_script.py

```diff
@@ -5,10 +5,10 @@
 from filelock import FileLock
 from retriever import datasets
 
-from status_dashboard_tools import get_dataset_md5
-from status_dashboard_tools import diff_generator
-from status_dashboard_tools import create_dirs
-from status_dashboard_tools import dataset_to_csv
+from .status_dashboard_tools import get_dataset_md5
+from .status_dashboard_tools import diff_generator
+from .status_dashboard_tools import create_dirs
+from .status_dashboard_tools import dataset_to_csv
```

Comment:
When importing using `.status_dashboard_tools` and running the script I get this error
![sss](https://user-images.githubusercontent.com/21103831/41912451-7418a57a-796d-11e8-9e50-4271317e4b4b.png)